lib/wasix/src/syscalls/journal/actions/close_thread.rs

```diff
@@ -0,0 +1,33 @@
+use super::*;
+
+impl<'a, 'c> JournalReplayRunner<'a, 'c> {
```

Comment:
This and the below files aren't syscalls, should they live in the syscalls directory?

Reply:
they replay the journal by doing syscalls

Reply:
ill rename them to make that easier to see